services/ingestion/etl/iati/csv/src/events/onParseCSV.js

```diff
@@ -53,7 +52,7 @@ export const handler = async (event, context, callback) => {
     (record, cb) => {
       try {
         const data = transformRecord(record);
-        return cb(null, `${JSON.stringify(data)}\n`);
+        return cb(null, `${JSON.stringify(data)}\r\n`);
```

Comment:
why? it was working fine until now, wasn't it?

Reply:
I'll remove this change